src/Protocol/SignatureHelp.php

```diff
@@ -29,4 +29,16 @@ class SignatureHelp
      * @var int|null
      */
     public $activeParameter;
+
+    /**
+     * @param SignatureInformation[] $signatures      The signatures.
+     * @param bool                   $activeSignature The active signature.
+     * @param bool                   $activeParameter The active parameter of the active signature.
+     */
+    public function __construct(array $signatures = [], int $activeSignature = 0, int $activeParameter = 0)
```

Comment:
Use `null` as default value for the last two params because they are optional on the object.